geth/jail/internal/timers/timers.go

```diff
@@ -79,11 +91,18 @@ func Define(vm *vm.VM, l *loop.Loop) error {
 
 		return otto.UndefinedValue()
 	}
-	vm.Set("clearTimeout", clearTimeout)
-	vm.Set("clearInterval", clearTimeout)
-	vm.Set("clearImmediate", clearTimeout)
+	err = vm.Set("clearTimeout", clearTimeout)
+	if err != nil {
+		return err
+	}
 
-	return nil
+	err = vm.Set("clearInterval", clearTimeout)
+	if err != nil {
+		return err
+	}
+
+	err = vm.Set("clearImmediate", clearTimeout)
+	return err
```

Comment:
Just `return vm.Set...` as you've already done in other places?

Reply:
In that case it's more clear for me. A lot of errchecks.